setup.py

```diff
@@ -1,18 +1,37 @@
 import codecs
+from os import path
 from setuptools import setup
 
 
+here = path.abspath(path.dirname(__file__))
+
+
+with open(path.join(here, 'README.rst'), 'r') as readme:
```

Comment:
It would appear that PyPI accepts a single format for the 'landing page' for humans: restructured text. By reusing the readme as the PyPI page, we'll only have to maintain one.